src/css/movie-summary.css

```diff
@@ -5,21 +5,35 @@
 	grid-template-columns: .10fr 1fr .10fr;
 }
 
+.about { 
+	color: white;
+}
+
+.movie-summary {
```

Comment:
like this new styling, much more readable/easier to uniformly render across movies.